src/components/newsletter/newsletter-sign-up.test.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { create } from "react-test-renderer";
+import NewsletterSignUp from "./newsletter-sign-up";
+import "@testing-library/jest-dom";
+import addToMailchimp from "gatsby-plugin-mailchimp";
+import { render, act, fireEvent, waitFor } from "@testing-library/react";
+
+jest.mock("gatsby-plugin-mailchimp");
+
+describe("NewsletterSignUp", () => {
+  it("renders correctly", () => {
+    const component = create(<NewsletterSignUp />).toJSON();
+    expect(component).toMatchSnapshot();
+  });
+
+  it("Name and email errors are not visible on render", async () => {
```

Comment:
I'd rename here to get the `it does x` verb phrase
`it("hides submission errors on render",…`